### PR TITLE
Allow Symfony 6

### PR DESCRIPTION
**Pull request by derrabus**
See also symfony/symfony#41301

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,16 +17,16 @@
...
+        "symfony/http-kernel": "~4.4 || ^5.0 || ^6.0",
         "monolog/monolog": "~1.22 || ~2.0"
     },
     "require-dev": {
-        "symfony/yaml": "~4.4 || ^5.0",
-        "symfony/console": "~4.4 || ^5.0",
-        "symfony/phpunit-bridge": "^5.1"
+        "symfony/yaml": "~4.4 || ^5.0 || ^6.0",
+        "symfony/console": "~4.4 || ^5.0 || ^6.0",
+        "symfony/phpunit-bridge": "^5.2"
```
Comment (jderusse) [inline composer.json:29]:
```suggestion
        "symfony/phpunit-bridge": "^5.2 || 6.0"
```
?

Comment (derrabus) [reply composer.json:29]:
```suggestion
        "symfony/phpunit-bridge": "^5.2 || ^6.0"
```

😉

**Review by jderusse — APPROVED** [review]

**Review by lyrixx — APPROVED** [review]

**Comment by Kocal** [comment]
Hi! What is the state of this PR?

I can't work on Symfony 6 support for SensioFrameworkExtraBundle :stuck_out_tongue:

**Comment by chalasr** [comment]
Thank you @derrabus.

**Pull request merged**